raspberryPI3/ros2_ws/src/detection/detection/obstacles_detection.py

```diff
@@ -0,0 +1,44 @@
+import rclpy
+from rclpy.node import Node
+
+from std_msgs.msg import Bool
+from interfaces.msg import Ultrasonic
+
+class ObstaclesDetection(Node):
+    
+    def __init__(self):
+        super().__init__('obstacles_detection')
+        self.publisher = self.create_publisher(Bool, 'obstacles_detection', 10)
+        self.get_logger().info('obstacles_detection_node started')
+        self.subscriber = self.create_subscription(Ultrasonic ,'us_data' , self.obstacle_detected, 10)
+        self.detection_distance=50 
```

Comment:
You can use ROS parameters with something like this :
`self.detection_distance = self.declare_parameter('detection_distance', 50).get_parameter_value().integer_value`

Reply:
Perfect  : 
![image](https://github.com/user-attachments/assets/78275fae-f0d6-4388-9d78-271de72ce519)

Reply:
Great